packages/wdio-browserstack-service/src/types.ts

```diff
@@ -3,10 +3,11 @@ import type { Options as BSOptions } from 'browserstack-local'
 
 export interface SessionResponse {
     // eslint-disable-next-line camelcase
-    automation_session: {
+    automation_session?: {
```

Comment:
Might be better to create two types of responses so that one is always required when the other is missing and vice versa, what do you think?

Reply:
Makes sense. Making those changes